app/grants/templates/grants/detail/tabs.html

```diff
@@ -69,6 +73,33 @@ <h4 class="m-auto text-center font-weight-semibold">{% trans "No Activity for th
       </div>
     {% endif %}
 
+    {% if tab == "stats" %}
+      <script src="https://www.gstatic.com/charts/loader.js"></script>
+      <script src="{% static "v2/js/grants/stats.js" %}"></script>
+      <div id=grant_stats_graph class="content-block content-block--white">
+        <div class="container">
+
+          <div class="value-container">
+            <div class="chart_container chart_container--big">
+              <p class="sub-title font-weight-semibold font-caption mb-1 pt-1">CONTRIBUTION HISTORY OVER TIME</p>
+              <div class="chart_container__content">
+                <div id="grant_chart"></div>
+              </div>
+              <div class="grant-status-list">
+                <p><span class="box box_1"></span> &nbsp;&nbsp;&nbsp;{% trans "Subscription Billing" %}</p>
+                <p><span class="box box_2"></span> &nbsp;&nbsp;&nbsp;{% trans "New Subscriptions" %}</p>
+                <p><span class="box box_3"></span> &nbsp;&nbsp;&nbsp;{% trans "One-Time Contributions" %}</p>
+                <p><span class="box box_5"></span> &nbsp;&nbsp;&nbsp;{% trans "CLR Matching Funds" %}</p>
+              </div>
+            </div>
+          </div>
+          <script>
+            document.max_graph = {{max_graph}};
+            document.history = {{history|safe}}; 
```

Comment:
Define it as a variable?

Reply:
hmm i was anchoring it off the document to give it global scope so i could reuse in the JS file.. let me see if that'll work still

Reply:
updated